src/main/notifications/index.ts

```diff
@@ -195,6 +199,17 @@ class NotificationManager {
         });
         this.restartToUpgradeNotification.show();
     }
+
+    private openNotificationPreferences() {
+        switch (process.platform) {
+        case 'darwin':
+            shell.openExternal('x-apple.systempreferences:com.apple.preference.notifications?Notifications');
+            break;
+        case 'win32':
```

Comment:
I don't think there's much value in that, unless we expect the values of those constants to change or if we need further verification.

Otherwise, `process.platform === 'win32'` is just as clear and concise as something like `Utils.isWindows()` would be